src/Microsoft.Graph/Models/Generated/TeamArchiveRequestBody.cs

```diff
@@ -26,7 +26,7 @@ public partial class TeamArchiveRequestBody
         /// Gets or sets ShouldSetSpoSiteReadOnlyForMembers.
         /// </summary>
         [JsonProperty(NullValueHandling = NullValueHandling.Ignore, PropertyName = "shouldSetSpoSiteReadOnlyForMembers", Required = Newtonsoft.Json.Required.Default)]
-        public bool? ShouldSetSpoSiteReadOnlyForMembers { get; set; }
```

Comment:
And this

Reply:
https://github.com/microsoftgraph/msgraph-metadata/commit/cf293dcc71b9336c294418ea609dc1b4e5d599a5#diff-e34459d76b3a635d414d3549ff656c4cL10530